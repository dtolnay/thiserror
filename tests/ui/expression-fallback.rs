use thiserror::Error;

#[derive(Error, Debug)]
#[error("".yellow)]
pub struct ArgError;

fn main() {}
