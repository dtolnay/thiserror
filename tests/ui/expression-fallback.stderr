error: expected `,`, found `.`
 --> tests/ui/expression-fallback.rs:4:11
  |
4 | #[error("".yellow)]
  |           ^ expected `,`

error: argument never used
 --> tests/ui/expression-fallback.rs:4:12
  |
4 | #[error("".yellow)]
  |         -- ^^^^^^ argument never used
  |         |
  |         formatting specifier missing

error[E0425]: cannot find value `yellow` in this scope
 --> tests/ui/expression-fallback.rs:4:12
  |
4 | #[error("".yellow)]
  |            ^^^^^^ not found in this scope
